src/main/java/seedu/address/logic/commands/EditGradeCommand.java

```diff
@@ -106,15 +108,15 @@ public boolean equals(Object other) {
         }
 
         // instanceof handles nulls
-        if (!(other instanceof AddGradeCommand)) {
+        if (!(other instanceof EditGradeCommand)) {
             return false;
         }
 
-        AddGradeCommand otherAddGradeCommand = (AddGradeCommand) other;
+        EditGradeCommand otherEditGradeCommand = (EditGradeCommand) other;
 
-        boolean sameAssignmentName = this.assignmentName.equals(otherAddGradeCommand.assignmentName);
-        boolean sameGrade = this.gradeString.equals(otherAddGradeCommand.gradeString);
-        boolean samePersonIndex = this.index.equals(otherAddGradeCommand.index);
+        boolean sameAssignmentName = this.assignmentName.equals(otherEditGradeCommand.assignmentName);
```

Comment:
maybe you want to add is in front of all these like eg. isSameAssignmentName? I think it shows more clearly you're looking at a boolean

Reply:
Yep, boolean expressions should be prefixed with "is"